mixin/mgd-php@1/example/CRM/BunnyDance.mgd.php

```diff
@@ -14,41 +14,40 @@
         'is_active' => TRUE,
         'is_reserved' => TRUE,
         'weight' => 1,
-        // FIXME: At time of writing, if I create a full CaseType with a definition, then the system runs out of memory.
-        // 'definition' => [
-        //   'activityTypes' => [
-        //     ['name' => 'Open Case', 'max_instances' => '1'],
-        //     ['name' => 'Follow up'],
-        //     ['name' => 'Change Case Type'],
-        //     ['name' => 'Change Case Status'],
-        //     ['name' => 'Change Case Start Date'],
-        //     ['name' => 'Link Cases'],
-        //     ['name' => 'Email'],
-        //     ['name' => 'Meeting'],
-        //     ['name' => 'Phone Call'],
-        //     ['name' => 'Nibble'],
-        //   ],
-        //   'activitySets' => [
-        //     [
-        //       'name' => 'standard_timeline',
-        //       'label' => 'Standard Timeline',
-        //       'timeline' => 1,
-        //       'activityTypes' => [
-        //         ['name' => 'Open Case', 'status' => 'Completed'],
-        //         ['name' => 'Phone Call', 'reference_offset' => '1', 'reference_select' => 'newest'],
-        //         ['name' => 'Follow up', 'reference_offset' => '7', 'reference_select' => 'newest'],
-        //       ],
-        //     ],
-        //   ],
-        //   'timelineActivityTypes' => [
-        //     ['name' => 'Open Case', 'status' => 'Completed'],
-        //     ['name' => 'Phone Call', 'reference_offset' => '1', 'reference_select' => 'newest'],
-        //     ['name' => 'Follow up', 'reference_offset' => '7', 'reference_select' => 'newest'],
-        //   ],
-        //   'caseRoles' => [
-        //     ['name' => 'Case Coordinator', 'creator' => '1', 'manager' => '1'],
-        //   ],
-        // ],
+        'definition' => [
+          'activityTypes' => [
+            ['name' => 'Open Case', 'max_instances' => '1'],
+            ['name' => 'Follow up'],
+            ['name' => 'Change Case Type'],
+            ['name' => 'Change Case Status'],
+            ['name' => 'Change Case Start Date'],
+            ['name' => 'Link Cases'],
```

Comment:
This one should be 'Eat furniture'

Reply:
Ah, I had a gnawing feeling that was the problem.

Reply:
lol